### DISCUSSION TITLE
有多少是真业主、有多少是卧底、有多少是吃瓜群众

### DISCUSSION THREAD

**Discussion by candlewill** [WeNeedHome/SummaryOfLoanSuspension#329] [General]
有多少是真业主、有多少是卧底、有多少是吃瓜群众

**Comment by daiyingwo** [comment]
吃瓜也犯法吗

**Comment by sqpk** [comment]
闹起来总归是好的，不然在沉默中只会灭亡，毕竟上头只管拿钱不管底层人死活

**Reply by qxf323** [reply]
大部分都是恒大啊  这个是之前的事情了，不要被这个利用了  说不定人家就是搅屎棍，美利坚的Dog

**Reply by edisom16** [reply]
> 大部分都是恒大啊  这个是之前的事情了，不要被这个利用了  说不定人家就是搅屎棍，美利坚的Dog

合着又是美方责任

**Comment by a793204032** [comment]
什么叫吃瓜群众？ 这事不是和每个人都息息相关，那天轮到你头上怎么办？

**Comment by remixzyb** [comment]
我不在乎什么卧底不卧底的 我只是觉得在这里活着没希望而已

**Reply by middleHallo** [reply]
有路子吗兄die我想跑路

**Reply by daydream-a** [reply]
想润+1

**Reply by kuro523** [reply]
说实话出国不难吧。。。

**Reply by ppwq2022** [reply]
> 

去越南泰国吧，应该不难

**Comment by guo897654050** [comment]
不管是否是业主，有一天你买房，这事落到你头上，你才来得及醒悟？难道大家还能盼着自己的祖国不好吗，都是被逼的。

**Reply by Murphy0Alan** [reply]
房地产苦人民久矣

**Comment by Coder-LinShun** [comment]
这一点也不重要

**Comment by Chain-Yu** [comment]
搞笑呢，有多少业主会上github？

**Reply by yalo-ddaa** [reply]
一个烂尾楼里面至少几百个人，几百人里面能没有程序员？

只要带起一个就一下午一传十，十传百。

**Reply by qxf323** [reply]
明显很多是以前，恒大不是很久的事情了吗？

**Reply by shadowlgd** [reply]
我是业主

**Comment by ethanrao** [comment]
这才多久，又开始分化群体，真是屡试不爽哈。

**Comment by Murphy0Alan** [comment]
卧底？我只能祝愿烂尾楼不会发生在你身上

**Comment by yalo-ddaa** [comment]
吃瓜群众怎么了，周围亲戚朋友里面打算买房，或者买过烂尾楼的很多啊。
我家人就有买过烂尾楼的，看到这个怎么就没用呢？

**Reply by qxf323** [reply]
大部分都是恒大啊  这个是之前的事情了，不要被这个利用了  说不定人家就是搅屎棍，美利坚的Dog

**Comment by Jesse3692** [comment]
![image](https://user-images.githubusercontent.com/24291671/178946483-3e891c66-aa6b-454a-b1fc-ca104875ae7b.png)

**Comment by jiaojiaozhe123456** [comment]
stand up

**Comment by zhangpc310** [comment]
如果你是真业主，我希望你是吃瓜群众；
如果你是卧底，我希望你是真业主；
如果你是吃瓜群众，我希望你是卧底

**Reply by eren-amundsen** [reply]
> 如果你是真业主，我希望你是吃瓜群众； 如果你是卧底，我希望你是真业主； 如果你是吃瓜群众，我希望你是卧底

套娃

**Comment by cccp057** [comment]
我来文档也被ban了，不用外网好像就这里可以看了

**Comment by white-cat** [comment]
我想说，肯定很多卧底！ 数据再跑今天，拉上排名了，估计还有某些外国记者猫着，看下怎么写一下新闻稿！
